server.js

```diff
@@ -1,236 +1,10 @@
-const _ = require('lodash');
-const jwt = require('jwt-simple');
-const expressStaticGzip = require('express-static-gzip');
-const express = require('express');
-const app = express();
+require('app-module-path').addPath(__dirname);
 
-const homebrewApi = require('./server/homebrew.api.js');
-const GoogleActions = require('./server/googleActions.js');
-
-// Serve brotli-compressed static files if available
-app.use('/', expressStaticGzip(`${__dirname}/build`, {
-	enableBrotli    : true,
-	orderPreference : ['br'],
-	index           : false
-}));
-
-process.chdir(__dirname);
```

Comment:
I tried to preserve this line during the rebase, but it still seems to be lost. @G-Ambatte, could you please advise on where should it be in context of refactored `server.js` file?

Reply:
From memory, it should proceed any calls to working directory specific resources, so the earlier the better. 
The simple test is to run `node server.js` from an alternate directory, i.e. with the current working directory as "C:/", run `node D:/Homebrewery/server.js`. If the line is in the correct place, Homebrewery will still run as normal.

Reply:
Thanks, restored this call in bfdde9104369dd0fc652a79ac6815a8ca60c5450. Tested locally: I'm able to run the server from an alternate directory like `node homebrewery/server.js`